libp2p/muxers/mplex/lpchannel.nim

```diff
@@ -21,6 +21,12 @@ export connection
 logScope:
   topics = "libp2p mplexchannel"
 
+when defined(libp2p_mplex_metrics):
```

Comment:
Actually, those are pretty cheap, we can enable them by default
Apart from that, LGTM

Reply:
2 of them are histograms. Still not a lot of metrics, but maybe we should keep them optional to avoid extra load on every message processing. Let me know, I can keep it as is or remove the `define`.